packages/base/src/tools.ts

```diff
@@ -384,7 +384,10 @@ export const openDatabase = () => {
  */
 export const saveToIndexedDB = async (
   key: string,
-  file: Blob,
+  file:
```

Comment:
Should the type be `any` here? We could save anything really.

Reply:
+1, sounds good. Done!